File: src/java/eu/sig/training/ch06/userservice/splitted/UserService.java
package eu.sig.training.ch06.userservice.splitted;

import java.util.ArrayList;
import java.util.List;

import eu.sig.training.ch06.userservice.User;
import eu.sig.training.ch06.userservice.UserInfo;

@SuppressWarnings("unused")
// tag::UserService[]
public class UserService {
    public User loadUser(String userId) {
        // ... 
        // end::UserService[]
        return new User();
        // tag::UserService[]
    }

    public boolean doesUserExist(String userId) {
        // ... 
        // end::UserService[]
        return true;
        // tag::UserService[]
    }

    public User changeUserInfo(UserInfo userInfo) {
        // ...
        // end::UserService[]
        return new User();
        // tag::UserService[]
    }

    public List<User> searchUsers(UserInfo userInfo) {
        // ... 
        // end::UserService[]
        return new ArrayList<User>();
        // tag::UserService[]
    }
}
// end::UserService[]

File: src/csharp/eu/sig/training/ch06/userservice/splitted/UserService.cs
using System.Collections.Generic;

namespace eu.sig.training.ch06.userservice.splitted
{

    // tag::UserService[]
    public class UserService
    {
        public User LoadUser(string userId)
        {
            // ...
            // end::UserService[]
            return new User();
            // tag::UserService[]
        }

        public bool DoesUserExist(string userId)
        {
            // ...
            // end::UserService[]
            return true;
            // tag::UserService[]
        }

        public User ChangeUserInfo(UserInfo userInfo)
        {
            // ...
            // end::UserService[]
            return new User();
            // tag::UserService[]
        }

        public IList<User> SearchUsers(UserInfo userInfo)
        {
            // ...
            // end::UserService[]
            return new List<User>();
            // tag::UserService[]
        }
    }
    // end::UserService[]
}
